src/Hotel.js

```diff
@@ -0,0 +1,13 @@
+class Hotel {
+	constructor() {
+
+	}
+
+	percentBooked(bookings, rooms, todaysDate) {
```

Comment:
I would consider renaming this method, because it returns the number of rooms booked for the day, rather than the percentage.

Reply:
Good call, I noticed that but didn't change it. I will on the next PR. Thanks for looking over this.